+bisos/+iteration/+step/transfer.m

```diff
@@ -0,0 +1,56 @@
+classdef transfer < bisos.iteration.Step
+   
+properties
+    type = 'transfer';
+    varout = [];
+    varin;
+    fun;
+end
+
+methods
+    function step = transfer(prob, output, fun_handle, input, varargin)
+        % New transfer step.
+        
+        assert(isa(fun_handle, "function_handle"), 'Not a function handle.');
+
+        step.fun = fun_handle; 
+        
+        assert(nargin(fun_handle)-1==length(input),...
+            'Number of inputs is incorrect %d\nShould be %d', length(input), nargin(fun_handle)-1);
```

Comment:
Please keep the message for the exception on a single line. Also, be a little more specific: Number of which inputs do you mean?